src/FreeAge/common/player.hpp

```diff
@@ -165,7 +167,9 @@ class Player {
   /// The player color index.
   /// TODO: should this be here?
   const int playerColorIndex;
-  
+
+  // TODO (maanoo): 
```

Comment:
Empty TODO?

Reply:
Just for future reference I write my name when there is something that must happen before the pr is merged.